flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/CatalogManager.java

```diff
@@ -1647,6 +1675,8 @@ private void execute(
                 command.execute(catalog.get(), objectIdentifier.toObjectPath());
             } catch (TableAlreadyExistException
                     | TableNotExistException
+                    | ModelNotExistException
+                    | ModelAlreadyExistException
```

Comment:
nits: 
ModelNotExistException -> ModelDoesNotExistException
ModelAlreadyExistException ->  ModelAlreadyExistsException

Reply:
with current situation I would vote for having naming similar to what we have for tables

>I am curious why we do not have view exceptions

Views are considered as tables